Update 22-to-30.md (#16537)
aspnet/AspNetCore.Docs: aspnetcore/migration/22-to-30.md

                new CamelCasePropertyNamesContractResolver());
  ```

### System.Runtime.Serialization attributes not supported

> <xref:System.Runtime.Serialization.DataContractAttribute> and <xref:System.Runtime.Serialization.DataMemberAttribute> are not supported in `System.Text.Json`. For more information, see the GitHub issue [System.Text.Json support to System.Runtime.Serialization](https://github.com/dotnet/corefx/issues/38758).

Reviewer: ```suggestion
The [[DataContract]](xref:System.Runtime.Serialization.DataContractAttribute) and [[DataMember]](xref:System.Runtime.Serialization.DataMemberAttribute) attributes aren't supported in `System.Text.Json`. For more information, see the GitHub issue [System.Text.Json support to System.Runtime.Serialization](https://github.com/dotnet/corefx/issues/38758).
```
